app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,10 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_tree .
```

Comment:
Have moved the generated active_admin.scss to vendor/assets/stylesheets now so it is only picked up when you visit /admin.